unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -79,7 +81,7 @@ export const MembersContent = ({ query }: MembersContentProps) => {
   const hasLocks = lockAddresses?.length > 0
 
   const onLockChange = (lock: Lock) => {
-    window.location.href = `/members?locks=${lock.address}`
```

Comment:
I am sorry but this is not the right approach. You should NOT change the location to change the state! 
The state should be set from the location OR programmatically via the UI itself.